api/settings.py

```diff
@@ -50,6 +50,15 @@
 RELENGAPI_AUTH = True
 SECRET_KEY = base64.b64decode(secrets['SECRET_KEY_BASE64'])
 
+# -- TASKCLUSTER --------------------------------------------------------------
+# it is important to be aware that even if we point to different taskcluster
+# instance, secrets which are loaded few lines above are still fetched from
+# production instance of taskcluster
+
+if APP_CHANNEL in ['staging']:
+    TASKCLUSTER_ROOT_URL = 'https://stage.taskcluster.nonprod.cloudops.mozgcp.net'
```

Comment:
Rather hardcode this based on channel, I think it would make more sense to put this value in the config.

Reply:
I was told that that configuration should be moved into settings.py. Either way it is fine by me.